package.json

```diff
@@ -65,7 +64,10 @@
 		"docker:update-core-unit-tests": "yarn docker:compose exec wordpress svn up /tmp/wordpress-develop/tests/phpunit/data/ /tmp/wordpress-develop/tests/phpunit/includes",
 		"docker:wp": "yarn docker:compose exec wordpress wp --allow-root --path=/var/www/html/",
 		"install-if-deps-outdated": "yarn check 2> /dev/null || yarn install --check-files --production=false --frozen-lockfile",
-		"lint": "eslint --ext .js,.jsx . _inc extensions modules",
+		"jetpack:build": "yarn jetpack:cd && yarn build-concurrently",
+		"jetpack:cd": "cd projects/plugins/jetpack",
+		"jetpack:lint": "yarn jetpack:cd && yarn lint",
```

Comment:
`jetpack:lint` is probably not needed.

We may not even need a `yarn lint` in Jetpack's package.json, since we should be linting at the monorepo level. I dropped all those kinds of scripts from Jetpack's composer.json already (which reminds me, I forgot to remove the references to them from Jetpack's package.json...).